models/org.go

```diff
@@ -50,7 +50,7 @@ func (org *User) GetOwnerTeam() (*Team, error) {
 }
 
 func (org *User) getTeams(e Engine) error {
-	return e.Where("org_id=?", org.ID).Find(&org.Teams)
+	return e.Where("org_id=?", org.ID).OrderBy("CASE WHEN name LIKE '" + OWNER_TEAM + "' THEN '' ELSE name END").Find(&org.Teams)
```

Comment:
This is not possible.

![image](https://cloud.githubusercontent.com/assets/1005086/20032711/cf130e98-a390-11e6-849f-03091bce8e31.png)